src/cleanup.js

```diff
@@ -4,16 +4,18 @@ import { join } from 'path';
 
 import { readdir, stat } from 'fs-extra';
 
+import type { LoggerType } from './types';
 import { tryRmrf } from './util';
 
 type CleanOptions = {|
     dir : string,
     interval : number,
     threshold : number,
-    onError : ?(mixed) => void
+    onError : ?(mixed) => void,
+    logger : LoggerType
 |};
 
-export function cleanDirectoryTask({ dir, interval, threshold, onError } : CleanOptions) : {| save : (string) => void, cancel : () => void |} {
+export function cleanDirectoryTask({ dir, interval, threshold, onError, logger } : CleanOptions) : {| save : (string) => void, cancel : () => void |} {
```

Comment:
I think we need to pass the logger in here as well: https://github.com/krakenjs/grabthar/blob/62e81fbe7ac1af99cd81988f7fbac1c2265fbfa9/src/poll.js#L186-L191

Reply:
oh nice!